includes/CodingStandards/GoogleSiteKit/Sniffs/Semantic/DescriptionStartsWithThirdPersonVerbSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Short description of every function and method docblock must start with a third person verb.
+ *
+ * @since n.e.x.t
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace PHP_CodeSniffer\Standards\GoogleSiteKit\Sniffs\Semantic;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Description Starts With Third Person Verb Sniff.
+ *
+ * @since n.e.x.t
+ */
+class DescriptionStartsWithThirdPersonVerbSniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return array( T_DOC_COMMENT_OPEN_TAG );
+	}
+
+	/**
+	 * Processes this sniff, when one of its tokens is encountered.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcs_file The current file being checked.
+	 * @param int                         $stack_ptr  The position of the current token in the
+	 *                                                stack passed in $tokens.
+	 * @return void
+	 */
+	public function process( File $phpcs_file, $stack_ptr ) {
```

Comment:
There's one particular instance where this should not apply, which is for constructor descriptions (`__construct`). Those (should) usually have the shape of simply `Constructor.` or `MyClass constructor.`.